client/client.go

```diff
@@ -673,17 +672,6 @@ func (c *Client) downloadMetaFromTimestamp(name string, m data.TimestampFileMeta
 	return b, nil
 }
 
-// decodeRoot decodes and verifies root metadata.
```

Comment:
Why is this function deleted now? Because it's unused?

Reply:
Yes